codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2StreamChannel.java

```diff
@@ -0,0 +1,979 @@
+/*
+ * Copyright 2019 The Netty Project
```

Comment:
This is basically the code that was present in `Http2MultiplexCodec.DefaultHttp2StreamChannel` so I can share it across the old (deprecated) implementation and the new `Http2MultiplexHandler`.